### DISCUSSION TITLE
Writing to a micrologix 1100 register using SLCDriver

### DISCUSSION THREAD

**Discussion by TagLinker** [ottowayi/pycomm3#298] [Q&A]
How to write to N7:0 register , reading is OK, but writing does not work.
The code use plc.write('N7:0', 100)

**Comment by akashkumarmandal** [comment]
Kindly provide the format in which i will perform write operation in SLCDriver.

Thanking you @ottowayi

**Reply by akashkumarmandal** [reply]
@ottowayi kindly share syntax to how to write data in SLCDriver
